app/views/students/_index.html

```diff
@@ -0,0 +1,107 @@
+<div ng-controller='StudentsIndexCtrl as controller'>
```

Comment:
- Update file's name to be `index.html` instead of `_index.html` as detailed above.
- This file is indented with tabs. Please fix this to be 2 spaces.
- Use single quotes when possible.